OpenBCI_32bit_Library.cpp

```diff
@@ -456,56 +437,47 @@ boolean OpenBCI_32bit_Library::processChar(char character) {
 
 /**
  * Start the timer on multi char commands
- * @param None
+ * @param cmd the command received on the serial stream. See enum MULTI_CHAR_COMMAND
```

Comment:
please add `{unsigned int}` and shouldn't this be a char? why is this four bytes instead of one? this should be `uint8_t` or `byte` or `char` i think

Reply:
Correct